### PR TITLE
Adds ticket serializer & ticket details to cell-ticket

### LINKED ISSUE #808: Add missing info on event table at admin/events & events
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

screenshot from 2017-08-26 12-08-45

### PR DESCRIPTION
**Pull request by geekyd**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds ticket serializer & ticket details to cell-ticket

Fixes #808

### CODE REVIEW

**Comment by sumedh123** [comment]
@geekyd Are you getting problem in adding because of no admin or co organizer rights?

**Comment by CosmicCoder96** [comment]
@geekyd You mean it is not possible to `include` it in the query right ?

**Comment by geekyd** [comment]
@CosmicCoder96 Nope, If we include the roles & the user does not have coorg. or org role, it will throw Error 500

**Comment by CosmicCoder96** [comment]
@geekyd But why will that scenario ever arise, this query will either be made by an admin , or a user who has access to his own events via `manage-events`?

**Comment by sumedh123** [comment]
@CosmicCoder96 Correct. @geekyd Did you try on local server?

**Comment by geekyd** [comment]
Ahh yes Thanks for pointing it out @CosmicCoder96, I kept on getting errors due to missing org role.
@sumedh123 Yes.

**Comment by geekyd** [comment]
@sumedh123 @CosmicCoder96 there is problem we cannot make a request using `include` for user/id/event for non admin users. The only possible way to do that will be to make individual requests to each event.

File: app/routes/events/list.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ export default Route.extend({
       ];
     }
     return this.get('authManager.currentUser').query('events', {
-      include      : 'event-topic,event-sub-topic,event-type,tickets,sessions,speakers',
+      include      : 'tickets,sessions,speakers,organizers,coorganizers,track-organizers,registrars,moderators',
```
Comment (geekyd) [inline app/routes/events/list.js]: @CosmicCoder96 @sumedh123 This will cause error, please check deployment link.

**Comment by niranjan94** [comment]
Deployment link: https://pr-819-open-event-frontend-fossasia.surge.sh

File: app/routes/events/list.js (hunk 1 above)
Comment (geekyd) [reply app/routes/events/list.js]: @sumedh123 @CosmicCoder96 Please check the deployment link.

**Comment by CosmicCoder96** [comment]
@geekyd The app will crash for non admin users, any workarounds?

**Comment by geekyd** [comment]
@CosmicCoder96 The only work around will be to either make changes to API or to make individual requests for each event using the Endpoint `events/id?include=oeganizer...`, which will not be possible using the ember-models-table.

**Comment by sumedh123** [comment]
@geekyd @CosmicCoder96 In this case, I think we should make the server guys change the API then.

**Comment by niranjan94** [comment]
This should be fixed on the server side. The permission schema should allow non-admins to access these things as long as they have permissions on an event level basis.

**Comment by geekyd** [comment]
@niranjan94 What should be done here?

**Comment by geekyd** [comment]
Okay I will open an issue related to this on the orga-server. Thanks

**Comment by niranjan94** [comment]
And the FE will also need to make requests correctly. 

Only organizers,coorganizers can see track-organizers,registrars,moderators.

track-organizers,registrars,moderators cannot see organizers,coorganizers,track-organizers,registrars,moderators 

This should be taken care of by FE

**Comment by geekyd** [comment]
@niranjan94 We are using `users/id/events` Endpoint for this route, I think that is already been taken care of, right?

**Comment by niranjan94** [comment]
No. Not at all. For example, a track-organizers or event A will get event A when you make a request to `users/id/events`. But he/she should not be able to see details about organizers,coorganizers can see track-organizers,registrars,moderators

**Comment by geekyd** [comment]
Okay got it, I will create an issue on the server and update the PR. Thanks

**Comment by geekyd** [comment]
@CosmicCoder96 @sumedh123 @harshitagupta30 Please review. Thanks!

**Comment by niranjan94** [comment]
Deployment link: https://pr-819-open-event-frontend-fossasia.surge.sh

**Pull request merged**